AprilTags/TagDetector.h

```diff
@@ -38,6 +38,7 @@ class TagDetector {
 	}
 	
 	std::vector<TagDetection> extractTags(const cv::Mat& image);
+	bool verifyQuad(std::vector<std::pair<float, float>> p, cv::Mat gray);
```

Comment:
1. const ref, beides
2. Ich verstehe die Schnittstelle nicht ganz. Es muss doch in irgendeiner Form die soll-id uebergeben werden - sonst kann das doch gar nicht geprueft werden?

Reply:
Der momentane Ansatz prüft ob überhaupt eine ID ausgelesen werden kann und das reicht auch bisher. Aber ja mit Vergleich ist es auf jeden Fall schöner.

Reply:
Also wegen mir kannst du die Funktion auch so aendern dass die einen int zurueckgibt: -1: kein quad gefunden.... sonst die id des erkannten quads...

Reply:
Ist alles eingebaut.